docs/data-sources/dns_records.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud_dns_records Data Source - terraform-provider-anxcloud"
+subcategory: ""
+description: |-
+  Provides DNS records for a specified zone.
+---
+
+# anxcloud_dns_records (Data Source)
+
+Provides DNS records for a specified zone.
+
+## Example Usage
+
+```terraform
+data "anxcloud_dns_records" "example" {
+  dns_zone = "example.com"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `zone_name` (String) Parent zone
+
+### Optional
+
+- `id` (String) The ID of this resource.
+
+### Read-Only
+
+- `records` (List of Object) List of known records on the zone (see [below for nested schema](#nestedatt--records))
+
+<a id="nestedatt--records"></a>
+### Nested Schema for `records`
+
+Read-Only:
+
+- `identifier` (String) DNS Record identifier. Changes on revision change and therefore shouldn't be used as reference.
```

Comment:
Should we expose the identifier at all with this limitation?

Reply:
Identifier is currently necessary for testing.